app/apollo/models/user.local.schema.js

```diff
@@ -323,9 +323,8 @@ UserLocalSchema.statics.getOrg = async function(models, me) {
   return org;
 };
 
-UserLocalSchema.statics.getOrgs = async function(context) {
+UserLocalSchema.statics.getOrgs = async function(models, me) {
```

Comment:
could we keep `getOrgs = async function(context)`, else we will need to fix IAM auth code and they may use other fields in the context.

Reply:
I updated the code to go back to passing in context